plugins/builder/osbuild.py

```diff
@@ -313,17 +313,18 @@ def handler(self, name, version, distro, image_types, target, arches, opts):
         self.logger.debug("Waiting for comose to finish")
         status = client.wait_for_compose(cid)
 
-        if not status.is_success:
-            self.logger.error("Compose failed: %s", str(status))
-            return {
-                'koji_builds': []
-            }
 
-        return {
-            'koji_builds': [bid],
-            'composer_id': cid,
-            'build': bid,
+        result = {
+            "status": status.status,
+            "osbuild": {
```

Comment:
For consistency, should we call this `composer` or `osbuild-composer`? Was discussing with @larskarlitski what the service and the domain should be called in the end, he suggested `composer.osbuild.org`, as it is strictly speaking not 1-to-1 with `osbuild` and we could imagine replacing the composer part in the future.

Reply:
I agree. It is changed.